lib/src/execution_context.dart

```diff
@@ -115,5 +115,7 @@ abstract class PostgreSQLResultRow implements List {
 ///
 /// Rows can be accessed through the `[]` operator.
 abstract class PostgreSQLResult implements List<PostgreSQLResultRow> {
+  /// How many rows did this query affect?
+  int get rowsAffected;
```

Comment:
I think `affectedRowCount` is a more precise name for this, e.g. we have `onlyReturnAffectedRowCount`, this aligns with that better.